MapboxNavigation/NavigationViewController.swift

```diff
@@ -687,6 +688,17 @@ extension NavigationViewController: NavigationServiceDelegate {
         return navigationComponents.allSatisfy { $0.navigationServiceShouldDisableBatteryMonitoring(service) }
     }
     
+    public func locationManagerDidChangeAuthorization(_ manager: CLLocationManager) {
+        let title = NSLocalizedString("ENABLE_PRECISE_LOCATION", bundle: .mapboxNavigation, value: "Enable precise location to  navigate", comment: "Label indicating precise location is off and needs to be turned on to navigate")
+        if #available(iOS 14.0, *), manager.accuracyAuthorization == .reducedAccuracy {
```

Comment:
Xcode 11 builds are [failing](https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-ios/1984/workflows/ceeb96e7-7c3a-498d-866f-030a7aa211d5/jobs/22062) due to references to `CLLocationManager.accuracyAuthorization`, which the iOS 13 SDK built into Xcode 11 doesn’t know about:

```
/Users/distiller/project/MapboxNavigation/NavigationViewController.swift:693:45: error: value of type 'CLLocationManager' has no member 'accuracyAuthorization'
        if #available(iOS 14.0, *), manager.accuracyAuthorization == .reducedAccuracy {
                                    ~~~~~~~ ^~~~~~~~~~~~~~~~~~~~~
/Users/distiller/project/MapboxNavigation/NavigationViewController.swift:693:71: error: cannot infer contextual base in reference to member 'reducedAccuracy'
        if #available(iOS 14.0, *), manager.accuracyAuthorization == .reducedAccuracy {
                                                                     ~^~~~~~~~~~~~~~~
```

The `#available` is important because it prevents this code from running on iOS 13, but that isn’t quite enough for the compiler. I ran into a similar problem when authoring #2649. Fortunately, we can apply two different workarounds:

* Because CLLocationManager is an Objective-C class, every property has a corresponding key path (as in KVO). Instead of getting `CLLocationManager.accuracyAuthorization` directly, call `value(forKey:)` to get the value for the key `accuracyAuthorization`.
* `CLAccuracyAuthorization.reducedAccuracy` is also new to the iOS 14 SDK, so #2649 implements an `MBNavigationAccuracyAuthorization` enumeration as a compatibility shim in Objective-C.

https://github.com/mapbox/mapbox-navigation-ios/blob/64dd90e79e882516d50e3577d854f80bbc1da2b8/MapboxNavigation/PassiveLocationManager.swift#L48-L50 https://github.com/mapbox/mapbox-navigation-ios/blob/8403eebf864d7693d61c59cfce5819f1e241b155/MapboxNavigation/CLLocationManager%2BMGLNavigationAdditions.h#L7